user_guide_src/source/conf.py

```diff
@@ -101,7 +101,7 @@
 #
 # Specifying a few options; just a starting point & we can play with it.
 html_theme_options = {
-
+    'display_version': False
```

Comment:
@jim-parry Perhaps this can be `True`? https://forum.codeigniter.com/thread-69043.html

Reply:
We are planning to visit user guide availability, across versions, before the 4.0 release.
That should be handled then, but not easily done using gh-pages.

Reply:
Most easy method that I think is creating folders with version numbers and uploading the file already builded. 

```
/user_guide/  # Current version
/user_guide/archive/ # List of versions
/user_guide/archive/4.0.2 # Version
```

But if I'm not mistaken, I've read an article on how to build gh-pages into sub-folders... which would automate the process.

Reply:
the problem isn't just gh-pages, but multiple versions in multiple repos and available in multiple forms (html, download, epub, pdf) ... leaving that for another day :)